src/Microsoft.Azure.ServiceBus/Core/IReceiverClient.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.ServiceBus.Core
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    public interface IReceiverClient : IClientEntity
```

Comment:
We are yet to completely finalize on all the APIs for clients as well as Receivers and Senders. Will take up documentation separately before releasing the package.